pallets/gear/src/manager.rs

```diff
@@ -85,14 +83,47 @@ pub enum HandleKind {
     Reply(H256, ExitCode),
 }
 
+pub struct ExtManager<T: Config> {
+    // Ids checked that they are users.
```

Comment:
Are these comments intended to be doc-comments?
```suggestion
    /// Ids checked that they are users.
```

Reply:
I don't think we need to doc comment private fields anywhere, so it's just a comments, but already adapted it, also added above ExtManager itself